New-SCOMHtmlNotificationChannel.ps1

```diff
@@ -0,0 +1,500 @@
+<#
+.SYNOPSIS
+    Creates SCOM notification channels with HTML formatting enabled.
+.DESCRIPTION
+    This script will create SCOM notification channels with HTML formatting options enabled. This allows for significantly richer notification templates.
+
+    Channels can either be created from scratch or clone their settings from an existing channel (and will therefore use it's predefined endpoints).
+
+    By default notification bodies will include appropriate links to the SCOM Web console, but by specifying the SquaredUpUrl parameter links will use Squared Up instead.
```

Comment:
"SquaredUpURL" as per parameter name, and perhaps prefix with a "-" to indicate it is a parameter.